src/styles/global.css

```
@@ -0,0 +1,1179 @@
+  display: grid;
+  gap: 0.22rem;
+}
+
+.contact-honeypot {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  margin: -1px;
+  padding: 0;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.2.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)